Cargo.toml

```diff
@@ -9,14 +9,19 @@ lto = true          # Enable Link Time Optimization
 codegen-units = 1   # Reduce number of codegen units to increase optimizations.
 panic = 'abort'     # Abort on panic
 
+[features]
+default = ["daemon"]
```

Comment:
hum, to be sure not to break anything, shouldn't `read-url` also be in the default (like it was before no?)

Reply:
my understanding is that these get merged — i.e `read-url` is still in default, since it is also in `daemon` (a cursory test by checking that `cargo run -- -i <some-url>` still works seems to confirm this)

Reply:
oh, sorry missed it in the daemon, fine for me